Drop-In/src/main/res/layout/bt_fragment_supported_payment_methods.xml

```diff
@@ -0,0 +1,94 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/bt_base_background"
+    android:minHeight="200dp"
+    tools:context="com.braintreepayments.api.DropInActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:id="@+id/bt_vaulted_payment_methods_wrapper"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:visibility="gone">
+
+            <RelativeLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingTop="@dimen/bt_choose_existing_payment_label_padding_top">
+
+                <Button
+                    android:id="@+id/bt_vault_edit_button"
+                    style="@style/Widget.AppCompat.Button.Borderless.Colored"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentEnd="true"
+                    android:text="@string/bt_edit"
+                    android:theme="@style/bt_edit_button"
+                    android:visibility="invisible" />
```

Comment:
Why is this set to invisible? Shouldn't there be an edit button in the top right?

<img width="374" alt="Screen Shot 2021-07-27 at 10 28 27 AM" src="https://user-images.githubusercontent.com/35243507/127182479-49512802-635a-4b84-8021-693a3828ca38.png">

Reply:
The edit button will display only when Vault Manager is enabled. We have to do it via the settings tab in Android.